generators/server/templates/gradle/_profile_dev.gradle

```diff
@@ -31,7 +31,7 @@ task gulpConstantDev(type: GulpTask) {
     args = ["ngconstant:dev", "--no-notification"]
 }
     <%_ } else { _%>
-task webpack(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task, dependsOn: '<%= clientPackageManager %>_install') {
+task webpackBuildDev(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task) {
```

Comment:
@deepu105 In JHipster gradle projects local versions can be installed with `-PnodeInstall` flag.  
Possible solution is to force local `node/npm/yarn` install also if `-Pwebpack` flag is specified.  
But to me it seems wrong. In what use case is someone using `-Pwebpack` while not `node` and `npm` or `yarn` installed?  
The purpose of `-Pwebpack` seems to be: if developer doesn't want to run development server with `yarn/npm start`, but changes front-end, then front-end can be compiled by `-Pwebpack` flag and run only `gradle/mvn` or am I missing something?  
One real world example for `-Pwebpack` use case is: running app after entity generation.

@atomfrede I didn't find these dependencies, can you give more information what did you mean with that?